tests/test_packages/test_connections/test_p2p_libp2p/base.py

```diff
@@ -73,7 +73,7 @@
 
 
 TIMEOUT = 20
-TEMP_LIBP2P_TEST_DIR = os.path.join(tempfile.gettempdir(), "tmp_libp2p_tests")
+TEMP_LIBP2P_TEST_DIR = tempfile.mkdtemp()
```

Comment:
when parallelizing tests this can lead to failure

`FileEsistsError: [Errno 17] File exists: 'tmp/tmp_libp2p_tests'`